README.md

```diff
@@ -304,7 +304,10 @@ This class extends [`EventEmitter`][] from Node.js.
     handling I/O in parallel.
   * `useAtomics`: (`boolean`) Use the [`Atomics`][] API for faster communication
     between threads. This is on by default. You can disable `Atomics` globally by
-    setting the environment variable `PISCINA_DISABLE_ATOMICS` to `1`.
+    setting the environment variable `PISCINA_DISABLE_ATOMICS` to `1`. 
+    If `useAtomics` is `true`, the one catch is that this stops asynchronous
+    operations that are still running from proceeding. Generally, tasks should
+    not spawn asynchronous operations without waiting for them to finish, though.
```

Comment:
```suggestion
    If `useAtomics` is `true`, it will cause to pause threads (stoping all execution) between tasks. Ideally, threads should wait for all operations to finish before returning control to the main thread (avoid having open handles within a thread).
```

Reply:
Thanks, done.